content/docs/user-guide/related-technologies.md

```diff
@@ -64,8 +64,7 @@ bringing best practices from software engineering into the data science field
 
 ## Workflow management systems
 
-Pipelines and dependency graphs
-([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) such as _Airflow_,
+Systems to manage data pipelines and [dependency graphs] such as _Airflow_,
```

Comment:
DAG is quite known term in the community, I think we are going a bit far removing it that aggressively (also may be bad for SEO)

Reply:
Agreed. See, for example, https://airflow.apache.org/docs/apache-airflow/stable/index.html.

Reply:
Yes I did intentionally remove it from most places except here. I thought the concept of "dependency graph" was much more informative and specific for us. I'll mention it again in a few more places then ⏳

Reply:
Done in ded42fff7.